narwhals/_pandas_like/dataframe.py

```diff
@@ -370,6 +372,10 @@ def drop_nulls(self, subset: str | list[str] | None) -> Self:
         plx = self.__narwhals_namespace__()
         return self.filter(~plx.any_horizontal(plx.col(*subset).is_null()))
 
+    def estimated_size(self, unit: SizeUnit) -> int | float:
+        sz = int(self._native_frame.memory_usage(deep=True).sum())
```

Comment:
why do we need to call `int` here?

Reply:
We don't. Forgot to remove an earlier experiment. 🧐

Reply:
@MarcoGorelli, should I let the result be np.Int? or convert to `int` (keep it on line 376)..

Reply:
numpy scalars are fine, they're pretty interoperable with python integers anyway

Reply:
Done. Thanks.